### ISSUE TITLE
CompileFast doesn't work with HasFlag

### ISSUE THREAD

**Issue by ABatenev** [dadhi/FastExpressionCompiler#374]
I have a flag enum, like
```
[Flags]
enum Foo
{
      None = 0,
      Black = 1,
      Red = 2,
      Green = 4,
      Blue = 8
}

class Bar
{
     Foo Foo
}
```
and i have expression

`Expression<Func<Bar, bool>>= new(x => x.Foo.HasFlag(Foo.Green ))`

After which I get the following error: System.ExecutionEngineException: "Exception_WasThrown"

**Comment by dadhi** [comment]
Ok, cool. Will check.

**Comment by zamaleev** [comment]
@dadhi sorry for bothering you, do you have any updates on this, can we some how help for investigation?

**Comment by dadhi** [comment]
I will look today, but no guarantee. Planned to do so anyway.

**Comment by zamaleev** [comment]
Thanks a mill!

**Comment by dadhi** [comment]
@zamaleev @ABatenev I have pushed the fix v4.0.1 to NuGet

**Comment by zamaleev** [comment]
thanks a mill @dadhi as always brilliant!
